docs/en/Samples/Index.md

```diff
@@ -62,6 +62,18 @@ While there is no Razor Pages & MongoDB combination, you can check both document
     * [Customize the SignIn Manager](https://community.abp.io/articles/how-to-customize-the-signin-manager-3e858753)
 * **GRPC Demo**: Shows how to add a gRPC service to an ABP Framework based web application and consume it from a console application.
   * [Source code](https://github.com/abpframework/abp-samples/tree/master/GrpcDemo)
+* **Telerik Blazor Integration**: Shows how to install and use Telerik Blazor components with the ABP Framework.
```

Comment:
The source code and Article Urls for the Telerik Blazor Integration article seem to be wrong.

Reply:
`Telerik Blazor Integration` article link fixed. Thanks :)